packages/sass-variables/_responsive.scss

```diff
@@ -0,0 +1,29 @@
+$breakpoints: (
+  // Portrait-mode phones are generally smaller than the 'small' breakpoint.
+    'small': 576px,
+  // Landscape-mode phones are between 'small' and 'medium'.
+    'medium': 768px,
+  // Tablets + smaller desktop browser windows are between 'medium' and 'large'.
+    'large': 1180px,
+  // Desktop browser windows + large landscape-mode tablets are larger than 'large'.
+) !default;
+
+@mixin respond-above($breakpoint) {
```

Comment:
Some examples out there use `respond-to` and `respond-from` for these types of mixins, but I found `respond-above` and `respond-below` clearer - totally willing to reconsider, though.

Reply:
I agree, the language you used seems clearer.